src/Plugin/ZynAddSubFX/ZynAddSubFX.cpp

```diff
@@ -282,7 +282,10 @@ class ZynAddSubFX : public Plugin
     */
     void loadProgram(uint32_t index) override
     {
-        setState(nullptr, defaultState);
+        for(int i=0; i < NUM_MIDI_PARTS; ++i) {
+            //set the program of the parts assigned to all midi channels
+            middleware->pendingSetProgram(i, index);
```

Comment:
`src/Output/DSSIaudiooutput.cpp` has similar code. However, in DPF, there seems to be no MIDI part(?), so I set the program to all parts. Probably not good?

Reply:
I'd stick to the assumption that the user expected behavior is to have only one instrument running in zyn when in plugin mode. So, load it once into the 0th program?

Reply:
You mean in plugin mode, there's only one part ("part 1") useful, so use only `middleware->pendingSetProgram(0, index);`?

Reply:
Technically the other parts are still there. If it wasn't more work to disable the extra parts it would likely make sense to do so in the plugin. Generally only the lowest index part is something I'd expect users to want to use.